tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.cs

```diff
@@ -29,6 +29,11 @@ public partial class Startup
         static Startup()
         {
             ManagedProfilerDirectory = ResolveManagedProfilerDirectory();
+            if (ManagedProfilerDirectory is null)
+            {
+                StartupLogger.Log("Managed profiler directory doesn't exist. Automatic instrumentation will be disabled");
```

Comment:
I would avoid using "managed profiler" in client-facing messages. "But I'm trying to use the tracer, not the profiler."

```suggestion
                StartupLogger.Log("Tracer home directory not found. Automatic instrumentation disabled.");
```

Reply:
lol, the line below...
```
Resolving managed profiler directory to:
```

Reply:
oops forgot to change. I'd love to put it in the next PR